otri/filtering/filters/split_filter.py

```diff
@@ -16,38 +15,70 @@ class SplitFilter(Filter):
     will be the last one (of index n+1).
     '''
 
-    def __init__(self, source_stream: Stream, key: Any, ranges: Sequence, ignore_none: bool = True, side: str = 'left'):
+    def __init__(self, input: str, output : Sequence[str], key: Any, ranges: Sequence, none_keys_output: str = None, side: str = 'left'):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'input'](https://app.codacy.com/gh/OTRI-Unipd/OTRI/pullRequest?prid=5649486)